src/templates/base.html

```diff
@@ -1,63 +1,75 @@
-        <link crossorigin="anonymous" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" rel="stylesheet">
-        <link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet"/>
-        <link href="/static/wp/css/grid.css?ver=4.7.3" id="grid-css" media="all" rel="stylesheet" type="text/css">
-        <link href="/static/wp/bootstrap/css/bootstrap.min.css" id="bootstrap-css" media="all" rel="stylesheet" type="text/css">
-        <link href="/static/wp/bootstrap/css/bootstrap-theme.css" id="bootstrap-theme-css" media="all" rel="stylesheet" type="text/css">
-        {% block style %}<link href="/static/style.css" rel="stylesheet">
-        {% endblock %}
-        <meta id="organized_list_items" data-name="{{ numCategories }}" data-other="{{other}}">
-        <link href="/static/wp/images/favicon.ico" rel="shortcut icon"/>
-    </head>
+    <meta charset="UTF-8">
+    <meta content="width=device-width, initial-scale=1.0" name="viewport">
+    <meta content="ie=edge" http-equiv="X-UA-Compatible">
+    <meta content="0.0.2" name="version">
+    <meta content="2019 Jan 08" name="revised">
+    {% block content_meta %}
+    <meta
+        content="KBase apps are analysis tools that you can use in KBase. Apps interoperate seamlessly to enable a range of scientific workflows (see figure below). Some of the apps were written by KBase scientists and developers; others are third-party tools that were integrated into KBase with our Software Developer Kit (SDK). The number of apps available in KBase will increase rapidly as members of the community use our SDK to integrate their analysis tools into the KBase platform.
+                                                                                                                                                                                                                                                                                                                                        The KBase App Catalog lists all of the currently available apps."
+        name="Kbase App Catalog">
+    {% endblock %}
+    {% block title %}
+    <title>KBase App Catalog</title>
+    {% endblock %}
+    <script src="{{ conf.ROOT_PREFIX }}/static/wp/js/jquery.js" type="text/javascript"></script>
+    <script src="{{ conf.ROOT_PREFIX }}/static/wp/js/jquery-migrate.min.js" type="text/javascript"></script>
+    <script src="{{ conf.ROOT_PREFIX }}/static/wp/js/kbaseUtils.js" type="text/javascript"></script>
+    <link crossorigin="anonymous" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css"
+        integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" rel="stylesheet">
+    <link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet" />
```

Comment:
Is this already included in the embedded Bootstrap, so we don't have to depend on the CDN?